cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/CentralUtils.java

```diff
@@ -164,4 +168,30 @@ public static String getCentralPackageURL(String org, String pkgName) {
         }
         return BALLERINA_CENTRAL_PRODUCTION_URL + "/" + org + "/" + pkgName;
     }
+
+    /**
+     * Create and get the home repository path.
+     *
+     * @return home repository path
+     */
+    public static Path createAndGetHomeReposPath() {
```

Comment:
Can't we use the `ProjectUtils.createAndGetHomeReposPath()`?

Reply:
Done